src/stats.c

```diff
@@ -249,6 +249,10 @@ void dump_stats(bool log, const char *cache_path) {
     snprintf(str, MAX_LINE_LEN, "  key2path:         %u", FETCH(filecache_key2path));
     print_line(log, fd, LOG_NOTICE, SECTION_FILECACHE_OUTPUT, str);
 
+    snprintf(str, MAX_LINE_LEN, "Notable Events:");
+    print_line(log, fd, LOG_NOTICE, SECTION_FUSEDAV_OUTPUT, str);
+    snprintf(str, MAX_LINE_LEN, "  conflicting 404:  %u", FETCH(conflicting_404));
```

Comment:
This seems incomplete.  Don't you need a `print_line` after `snprintf`?

Reply:
Quite right. Fixed.